homeassistant/components/blebox/const.py

```diff
@@ -23,6 +29,14 @@
 UNSUPPORTED_VERSION = "unsupported_version"
 UNKNOWN = "unknown"
 
+
+BLEBOX_TO_CLIMATE_HVAC_MODE = {None: None, False: HVAC_MODE_OFF, True: HVAC_MODE_HEAT}
+BLEBOX_TO_CURRENT_HVAC_MODE = {
+    None: None,
```

Comment:
I don't think None should be in these mappings, as it is not a mode.

Reply:
This was mostly a result of refactoring if/else logic into a map, and then moving the map to consts.

I think the only way to get rid of all the "None" cases if the device is set to "unavailable" ("unknown?") after initialization (and before the first successful update). And if, in that case, the state properties would be guaranteed not to be called on an unavailable entity. (Or something like that).

Then again, there's more flexibility if individual state attributes can be "None" at any time. (Which might make sense in practice - e.g. partially broken/unsupported features, error states, etc).

Another idea is to clear the initial `SUPPORT_*` attributes until the first successful connection with the device. But this might have other side effect in HA that I don't know about...

Reply:
Anyway, it's fixed.